### PR TITLE
Add Gitter and Slack buttons on the homepage

### PR DESCRIPTION
**Pull request by revolter**
Related to https://github.com/sqlitebrowser/sqlitebrowser/pull/1417.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -299,6 +299,18 @@ <h2>
 
   <p>This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the Architect theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.</p>
 
+  <a href="https://gitter.im/sqlitebrowser/sqlitebrowser?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge">
```
Comment (justinclift) [inline index.html]: We should probably leave off the tracking variables (`utm_source`, etc). :smile:

Comment (revolter) [reply index.html]: Should we also remove them from the README?

Comment (justinclift) [reply index.html]: Oh, didn't notice them there.  Yep, definitely. :smile:

**Comment by justinclift** [comment]
You're welcome to push simple changes directly to master btw.  Just saying. :wink:

**Comment by revolter** [comment]
I know, but another pair of eyes is always good. Also, I did had a mistake (tracking attributes).

**Comment by justinclift** [comment]
No worries. :smile:

File: index.html (hunk 1 above)
Comment (revolter) [reply index.html]: Done

**Pull request merged**